javascript-functions/main.js

```diff
@@ -0,0 +1,29 @@
+function convertMinutesToSeconds(minutes) {
+  return (minutes * 60);
+}
+console.log('2 Mins is equal to: ', convertMinutesToSeconds(2) + ' seconds.');
+
+function greet(name) {
+  return 'Hey, ' + name;
+}
+console.log('Greeting: ', greet('Cody') + '.');
+
+function getArea(width, height) {
+  return (width * height);
+}
+console.log('Area is equal to: ', getArea(17, 42) + ' units squared.');
+var person = {
+  firstName: 'Phillip',
+  lastName: 'Lam'
+};
+function getFirstName(person) {
+  return (person.firstName);
+}
+console.log('First name is: ', getFirstName(person));
+
+var array = ['propane', 'and', 'propane', 'accessories'];
+var lastIndex = array.length - 1;
```

Comment:
You should be doing this inside of the function block. Otherwise, the function will only ever return the value at the index saved in `lastIndex`, which would not be the end for every single array.